### PR TITLE
Make the URL passed to add_query_arg safe

### PR DESCRIPTION
**Pull request by philipjohn**

### CODE REVIEW

File: liveblog.php [hunk 1]
```diff
@@ -633,7 +633,7 @@ private static function add_default_plupload_settings() {
 	private static function get_entries_endpoint_url() {
 		$post_permalink = get_permalink( self::$post_id );
 		if ( false !== strpos( $post_permalink, '?p=' ) )
-			$url = add_query_arg( self::url_endpoint, '', $post_permalink ) . '='; // returns something like ?p=1&liveblog=
+			$url = add_query_arg( self::url_endpoint, '', rawurlencode( $post_permalink ) ) . '='; // returns something like ?p=1&liveblog=
```
Comment (nickdaugherty) [inline liveblog.php]: The 3rd param to `add_query_arg()` is the url to modify (unless you pass an array as first param)...it doesn't actually need url encoded, just the individual params. In this case, it's just an empty string, so is safe as-is.

Comment (philipjohn) [inline liveblog.php]: Damn. Then I have mis-interpreted @simonwheatley's summary of the issue.

Comment (philipjohn) [inline liveblog.php]: I read it again and understand now

Comment (nickdaugherty) [inline liveblog.php]: Alright cool, no worries.

**Pull request merged**